### PR TITLE
Recipe missing ingredient

### PR DESCRIPTION
**Pull request by syboulette**

### CODE REVIEW

File: app/controllers/lists_controller.rb [hunk 1]
```diff
@@ -1,68 +1,35 @@
...
-
-    if @list.save
-      redirect_to lists_path
-    else
-      render :new, status: :unprocessable_entity
-    end
-  end
-
-  def destroy
-    authorize @list
```
Comment (akramerin) [inline app/controllers/lists_controller.rb:35]: why is the destroy method deleted, if the before_action is referring to it?

File: app/views/recipes/show.html.erb [hunk 2]
```diff
@@ -26,6 +26,41 @@
...
+            <% if fridge_ingredient.nil? %>
+              <% fridge_ingredient = FridgeIngredient.new(ingredient_id: ingredient_id, quantity: 0) %>
+            <% end %>
+            <% if fridge_ingredient.quantity < @recipe_ingredients.find_by(ingredient_id: ingredient_id).quantity %>
+              <% missing_ingredients << Ingredient.find(ingredient_id) %>
+            <% end %>
+          <% end %>
+          <% if missing_ingredients.empty? %>
+            <h3>You have everything for your recipe!</h3>
+            <p>If you want to make your recipe now, we can automatically remove the ingredietn fromyour fridgi by clicking the link below!</p>
```
Comment (akramerin) [inline app/views/recipes/show.html.erb:45]: I now, tiny stuff, but please fix spelling errors

(remove the ingredient from you Fridgi by clicking the link below!)

File: app/models/user.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   has_one :fridge, dependent: :destroy
-  has_many :list, dependent: :destroy
+  has_one :list, dependent: :destroy
```
[the remark this answers is not in the document]
Comment (D-Zaleski) [reply app/models/user.rb:7]: agreed

File: app/views/recipes/show.html.erb (hunk 2 above)
Comment (D-Zaleski) [reply app/views/recipes/show.html.erb:45]: Important details

File: app/controllers/lists_controller.rb (hunk 1 above)
Comment (syboulette) [reply app/controllers/lists_controller.rb:35]: it is not :) and we have a permanent List now so cannot delete it

Comment (akramerin) [reply app/controllers/lists_controller.rb:35]: i realized that later on in the code :) sweet

**Review by D-Zaleski — COMMENTED** [review]
I don't see any html class conflicts with mine. So, good

**Pull request merged**